packages/store/operators/src/compose.ts

```diff
@@ -1,7 +1,16 @@
 import { StateOperator } from '@ngxs/store';
 
-export function compose<T>(...operators: StateOperator<T>[]) {
```

Comment:
![image](https://user-images.githubusercontent.com/7337691/59137065-8e767500-898e-11e9-8a5e-55a0ba119d5c.png)

Reply:
Yeah, this build is correct for me also, but not working for test